cmd/lncli/cmd_payments.go

```diff
@@ -1297,6 +1297,12 @@ var forwardingHistoryCommand = cli.Command{
 			Name:  "max_events",
 			Usage: "the max number of events to return",
 		},
+		cli.BoolFlag{
+			Name: "skip_peer_alias_lookup",
+			Usage: "allows to skip the peer alias lookup " +
```

Comment:
nit: I think "allows to skip" sounds a bit strange. Perhaps rather just "skip the peer alias lookup ..."